api/nakadi-event-bus-api.yaml

```diff
@@ -1567,13 +1651,14 @@ definitions:
           future there could be others.
       schema:
         type: string
-        $ref: '#/definitions/EventTypeSchema'
         description: |
           The schema as string in the syntax defined in the field type. Failure to respect the
           syntax will fail any operation on an EventType.
 
-          To have a generic, undefined schema it is possible to define the schema as `"schema":
-          "{\"additionalProperties\": true}"`.
+          When defining a schema, every property should be explicitly defined with its type. It means that,
+          additionalProperties true is not accepted.
```

Comment:
We have a need to have 2 constructs that may have problems here - can you confirm that we will be able to work with this, or are we effected.

Firstly, we use `additionalProperties` to define a `Map[String, SomeDefinition]`:

```json
       "some_map": {
          "additionalProperties": {
            "$ref": "#/definitions/SomeOtherDefinition"
          }
        }
```

Secondly, we need to have a property within the event (deep nested in fact), that has a type of simply `object` - the value of this property is a well formatted JSON object, the contents of which are fully validated prior to event submission, but this content depends on a flexible data model that can change over time (however, we also have careful compatible change control within this own data modelling construct.)

Actually, the "property" mentioned is in several definitions, and is handled by having the equivalent of the following definition "mixed" in with other definitions (using `allOf` composition): 

```json
"FlexibleModelledData": {
      "type": "object",
      "required": [
        "data_modeled_and_validated_elsewhere"
      ],
      "properties": {
        "data_modeled_and_validated_elsewhere": {
          "type": "object"
        }
      }
```